core/themes/seven/js/script.js

```diff
@@ -0,0 +1,324 @@
+            var $activeTabText = $('<span class="expand-dropdown-tabs-label">' + $('li.active a', $primaryTabs).html() + '</span>');
+            $('.element-invisible', $activeTabText).remove();
+            $('.expand-dropdown-tabs-label', $tabsWrapper).replaceWith($activeTabText);
+            $('.expand-dropdown-tabs-control', $tabsWrapper).css('left', 'auto');
+
+            // Cleanup things that may have been left over from other
+            // responsive tab strategies.
+            $('.responsive-tabs-dropdown', $tabsWrapper).remove();
+            $primaryTabs.css('padding-left', 0);
+          }
+
+          // Insert $responsiveTabsDropdown to markup if it's not empty.
+          if ($('li', $responsiveTabsDropdown).length > 0) {
+            $('.duplicated-tab', $responsiveTabsDropdown).removeClass('duplicated-tab');
+            if ($('.responsive-tabs-dropdown').length > 0) {
+              $('.responsive-tabs-dropdown', $tabsWrapper).replaceWith($responsiveTabsDropdown);
+            }
+            else {
+              $primaryTabs.after($responsiveTabsDropdown);
+            }
+          }
+        }
+      }
+    }
+
+    // If they click outside of the responsive tabs, shut them
+    $('html').click(function(e){
+      var $target = $(e.target);
+      if (responsiveTabs && !$target.is('.responsive-tabs-processed') && $target.parents('.responsive-tabs-processed').length < 1) {
+        console.log('DOIN STUFF');
+        closeTabsDropdown();
+      }
+    });
+
+    /**
+     * Check to see when webfont has loaded and adjust the tabs display
+     */
+    var checkFontCounter = 0;
+    // Append an invisible element that will be monospace font or our desired
+    // font. We're using a repeating i because the characters width will
+    // drastically change when it's monospace vs. proportional font.
+    var $checkFontElement = $('<span id="check-font-wrapper"><span id="check-font">iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii</span></span>');
+    $('body').append($checkFontElement);
+
+    // Function to check the width of the font, if it's substantially different
+    // we'll know we our real font has loaded
+    function checkFont() {
+      var currentWidth = $checkFontElement.width();
+      if (currentWidth < 200 || checkFontCounter >= 60) {
+        // If our font has loaded, or it's been 6 seconds
+        adjustTabsDisplay();
+        // Clean up after ourselves
+        clearInterval(checkFontInterval);
+        $checkFontElement.remove();
+        calculateTabWidths();
+      }
+      checkFontCounter++;
+    }
+    var checkFontInterval = setInterval(checkFont, 100);
+
+    // Resource friendly event handler from MDN
+    var throttle = function(type, name, obj) {
+      obj = obj || window;
+      var running = false;
+      var func = function() {
+        if (running) { return; }
+        running = true;
+        requestAnimationFrame(function() {
```

Comment:
Looks like requestAnimationFrame() may throw function not found on <IE10: https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame#Browser_compatibility

If needing a throttle, perhaps the one from Admin Bar would do (where `adjustItems` would be replaced in this case with `handleResize`):

```
  // Re-adjust items when window is resized.
  var resizeTimeout;
  $(window).on('resize.adminBar', function (event) {
    clearTimeout(resizeTimeout);
    resizeTimeout = setTimeout(adjustItems, 50);
  });
```